core/src/test/kotlin/fr/sncf/osrd/pathfinding/PathPropEndpointTest.kt

```diff
@@ -17,25 +18,18 @@ class PathPropEndpointTest : ApiTest() {
 
     @Test
     fun simpleSmallInfraTest() {
-        val trackSectionRanges =
-            listOf(
-                DirectionalTrackRange(
```

Comment:
I had to rewrite all the tests in this file, there was no easy way to get a fully valid path from just track ranges. Especially as the ranges previously used were not contiguous.

The paths are now defined in terms of routes, I've double-check all results by hand to make sure they actually match the infra data.